### ISSUE TITLE
Compile sass file to css

### ISSUE THREAD

**Issue by valdoryu** [codesthq/vuelendar#10]
Having both css and sass would allow users without sass-loader to use the package and sass users to override variables to custom style the component

**Comment by lusarz** [comment]
That's true, I'm wondering how to setup it technically. Just adding `css/vuelendar.css` file and remember to rebuild it manually everytime we'll change something in `scss` ?

**Comment by valdoryu** [comment]
Yes, I think creating a npm command to compile the scss to css is the only easy way to do. However we need to add it to git ignore to avoid repository pollution with compiled files
